### PR TITLE
Exclude io.zipkin.zipkin2:zipkin test-jar (1.1.x)

### PR DESCRIPTION
**Pull request by meltsufin**
Fixes: #1725.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -95,6 +95,12 @@
 				<groupId>io.zipkin.gcp</groupId>
 				<artifactId>zipkin-sender-stackdriver</artifactId>
 				<version>${zipkin-gcp.version}</version>
+				<exclusions>
+					<exclusion>
+						<groupId>io.zipkin.zipkin2</groupId>
+						<artifactId>zipkin</artifactId>
+					</exclusion>
```
Comment (dzou) [inline pom.xml]: Do you also need to add the `<type>` here? Like `<type>test-jar</type>`?

Comment (meltsufin) [reply pom.xml]: Doesn't look like it.

Comment (dzou) [reply pom.xml]: Isn't there a core artifact also named 'zipkin' though? https://repo.maven.apache.org/maven2/io/zipkin/zipkin2/zipkin/2.12.4/zipkin-2.12.4.pom

How does it disambiguate between the 'zipkin' core and the zipkin test jar?

Comment (meltsufin) [reply pom.xml]: `<type>` is not allowed under `dependencyManagement`.
In any case, `zipkin-sender-stackdriver` doesn't depend on `zipkin` core.
So, we should be OK. Other deps will bring it in.

**Comment by meltsufin** [comment]
@artembilan In this branch, I'm not sure exactly why, but it's not enough to exclude it in dependency management. It's still being pulled in.

**Comment by artembilan** [comment]
>  It's still being pulled in.

Can you print `dependency:tree` to be sure who brings that dependency for us?

**Comment by meltsufin** [comment]
I think I have to go back to excluding it everywhere. It's actually the case on both branches. My initial test must've been incorrect.
```
[INFO] |  +- io.zipkin.gcp:zipkin-sender-stackdriver:jar:0.9.0:compile
[INFO] |  |  +- io.zipkin.gcp:zipkin-translation-stackdriver:jar:0.9.0:compile
[INFO] |  |  +- io.zipkin.reporter2:zipkin-reporter:jar:2.7.13:compile
[INFO] |  |  +- com.google.api.grpc:grpc-google-devtools-cloudtrace-v1:jar:0.1.1:compile
[INFO] |  |  |  \- com.google.api.grpc:grpc-google-common-protos:jar:1.16.0:compile
[INFO] |  |  +- io.zipkin.zipkin2:zipkin:test-jar:tests:2.11.11:compile
```

**Pull request merged**